.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["next", "next/core-web-vitals", "prettier"],
+  "extends": ["next/core-web-vitals", "prettier"],
   "rules": {
     "@next/next/no-img-element": "off"
```

Comment:
@atapas  Just a suggestion, can we also add 
```
react-hooks/rules-of-hooks": "error",
react-hooks/exhaustive-deps": "warn",
```
for component re-rendering benefits.
refs: [React Hooks](https://reactjs.org/docs/hooks-rules.html)

Reply:
We can but we should do it in separate PR. We should keep PR to fixing one single thing. It will make things easier.